### PR TITLE
add some demo features

### PR DESCRIPTION
**Pull request by botany-demo**
add some demo features

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -47,3 +47,5 @@ update auth flow
 more documentation
 
 add new user settings table
+
+add some demo features
```
Comment (botany-lead) [inline readme.md:51]: great feature 👍

Comment (botany-lead) [reply readme.md:51]: hey can you look at this @botany-demo

Comment (botany-demo) [reply readme.md:51]: looks good to me

**Review by botany-lead — APPROVED** [review]
this is ready to go :shipit:

**Comment by botany-lead** [comment]
testing @botany-demo

**Pull request merged**